src/main/java/edu/harvard/iq/dataverse/harvest/server/web/servlet/OAIServlet.java

```diff
@@ -69,6 +70,18 @@ public class OAIServlet extends HttpServlet {
     
     @EJB
     SystemConfig systemConfig;
+
+    @Inject
+    @ConfigProperty(name = "dataverse.oai.server.maxidentifiers", defaultValue="100")
+    private Integer maxListIdentifiers;
+    
+    @Inject
+    @ConfigProperty(name = "dataverse.oai.server.maxsets", defaultValue="100")
+    private Integer maxListSets;
+    
+    @Inject
+    @ConfigProperty(name = "dataverse.oai.server.maxrecords", defaultValue="10")
+    private Integer maxListRecords;
```

Comment:
**praise:** All hail to mighty MPCONFIG!

(Might be refactored with what is in #8823, but it's a great start!!! :partying_face: )